src/monorepo-shipit/config/sx-tailwind-website.js

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+
+import type { ConfigType } from '../ConfigType.flow';
+
+module.exports = ({
+  getStaticConfig() {
+    return {
+      repository: 'git@github.com:adeira/sx-tailwind-website.git',
```

Comment:
Can you please create the repository first?

Reply:
Done